src/main/java/frc/trigon/robot/commands/Commands.java

```diff
@@ -70,4 +74,17 @@ public static Command duplicate(Command command) {
                 command.getRequirements().toArray(Subsystem[]::new)
         );
     }
+
+    public static Command getClimbCommand() {
+        return new SequentialCommandGroup(
+                new InstantCommand(
+                        () -> {
+                            CommandConstants.IS_CLIMBING = true;
+                            Logger.recordOutput("IsClimbing", true);
+                        }
+                ),
+                ClimberCommands.getSetTargetStateCommand(ClimberConstants.ClimberState.PREPARE_FOR_CLIMB).until(OperatorConstants.CONTINUE_TRIGGER),
```

Comment:
@levyishai maybe I could have it be until it reaches the target state instead of until the trigger?